Sming/Services/libb64/cencode.c

```diff
@@ -19,11 +19,12 @@ void base64_init_encodestate(base64_encodestate* state_in)
 char base64_encode_value(char value_in)
```

Comment:
Please leave the coding style of the files under Sming/Services/libb64 intact. This way we can easier track changes with the upstream version of libb64 and update when needed.